Move db queries from controller to repository

christopher-timm opened the pull request.

christopher-timm committed:
Add model/repo for tokens

christopher-timm committed:
Move db queries to repository

christopher-timm committed:
Remove comments

christopher-timm committed:
Codacy fixes

albig commented on Classes/Controller/CalendarController.php:
```diff
@@ -27,6 +27,16 @@ class CalendarController extends AbstractController
      */
     protected $allIssues = [];
 
+    protected $docucmentRepository;
```
```suggestion
    protected $documentRepository;
```

albig reviewed (changes requested):
Please check and fix all Scrutinizer hints.

christopher-timm committed:
Fix typo in CalendarController

Co-authored-by: Alexander Bigga <alexander.bigga@slub-dresden.de>

albig commented on Classes/Controller/CalendarController.php:
```diff
@@ -227,30 +220,18 @@ public function yearsAction()
-            )
-            ->orderBy('tx_dlf_documents.mets_orderlabel')
-            ->execute();
+        $documents = $this->docucmentRepository->getChildrenOfYearAnchor($this->doc->cPid, $this->doc->uid, 'year');
```
```suggestion
        $documents = $this->documentRepository->getChildrenOfYearAnchor($this->doc->cPid, $this->doc->uid, 'year');
```

Commit:
Update Classes/Controller/CalendarController.php

christopher-timm committed:
Scrutinizer fixes

Commit:
Fix errors in oai-pmh plugin

Commit:
Fix verbListSets

Commit:
Cleanup verbListIdentifiers

Commit:
Fix verbListMetadataFormats

Commit:
Fix verbIdentify

Commit:
Fix verbIdentify again

Commit:
Remove pages settings from oai-pmh plugin

Commit:
Fix ListRecords again

Commit:
Cache only non-empty result sets

Commit:
Refactoring token usage i oai-pmh plugin

Commit:
Use more Extbase in feeds plugin

Commit:
Use the actionLog repository

Commit:
Fix basket

Commit:
Change calendar plugin

Commit:
Remove pages setting from calendar plugin

Commit:
Fixes and small refactoring of collection plugin

Commit:
Remove unused settings

Commit:
Little refactoring the statistics plugin

Commit:
Simplify flexform

christopher-timm committed:
Merge pull request #5 from albig/chrizzor-dev-extbase-fluid-queries

Fixes of the extbase integrations - work in progress

Commit:
Move class mapping to global ext_typoscript_setup.txt

- use documentRepository in Document class
- change getInstance() to use settings over pid only as parameter

Commit:
Small improvements in search controller / configuration

Commit:
Cleanup MetadataRepository

christopher-timm committed:
Merge pull request #6 from albig/chrizzor-dev-extbase-fluid-queries

Move class mapping to global ext_typoscript_setup.txt
